docs/code-search/queries/index.mdx

```diff
@@ -28,29 +25,13 @@ Keyword search matches individual terms anywhere in the document or the filename
 | [`foo OR bar`](https://sourcegraph.com/search?q=context:global+foo+OR+bar&patternType=keyword)                                            | Match documents containing `foo` _or_ `bar` anywhere in the document.                                                            |
 
 <Callout type="tip">Matching is case-insensitive by default (toggle the `Aa` button to change).</Callout>
+<Callout type="note">In version 5.4, `keyword` replaced `standard` as the new default pattern type.</Callout>
 
-<Callout type="tip">Keyword search can be enabled for a Batch Change by adding [patternType:keyword](/code-search/queries/language#pattern-type) to the query in [a batch spec](/batch-changes/quickstart).</Callout>
-
-<Callout type="tip">[patternType:keyword](/code-search/queries/language#pattern-type) is supported in [Code Monitors](/code_monitoring/quickstart) and [Code Insights](/code_insights/quickstart). When creating a monitor or an insight from a search, it will be automatically populated with `patternType:keyword`. When building one by hand, be sure to include `patternType:keyword` if you want keyword search.</Callout>
-
-### Standard search
```

Comment:
Removed the "Standard search" and "Structural search" sections to keep these high-level docs simple

Reply:
It feels very good to remove these sections here, but maybe we should keep structural search and move it somewhere else?

Reply:
never mind, I saw we still have a dedicated page for structural search.